presto-parser/src/main/antlr4/com/facebook/presto/sql/parser/SqlBase.g4

```diff
@@ -260,6 +260,7 @@ primaryExpression
     : NULL                                                                           #nullLiteral
     | interval                                                                       #intervalLiteral
     | identifier STRING                                                              #typeConstructor
+    | DOUBLE_PRECISION STRING                                                        #typeConstructor
```

Comment:
Maybe it would be good to extract `typeConstructor`? E.g.:
```
typeConstructor
      : indentifier STRING
      | DOUBLE_PRECISION STRING
```

@haozhun, thoughts?